packages/thermal-utils/src/generate-barcode-svg.ts

```
@@ -5,9 +5,49 @@
+	encode: (opts: RenderOptions) => string;
+}
+
+const BARCODE_SPECS: Record<string, BarcodeSpec> = {
+	code128: { bcid: 'code128', encode: (opts) => code128(opts, drawingSVG()) },
+	code39: { bcid: 'code39', encode: (opts) => code39(opts, drawingSVG()) },
+	code93: { bcid: 'code93', encode: (opts) => code93(opts, drawingSVG()) },
+	ean13: { bcid: 'ean13', encode: (opts) => ean13(opts, drawingSVG()) },
+	ean8: { bcid: 'ean8', encode: (opts) => ean8(opts, drawingSVG()) },
+	upca: { bcid: 'upca', encode: (opts) => upca(opts, drawingSVG()) },
+	upce: { bcid: 'upce', encode: (opts) => upce(opts, drawingSVG()) },
+	codabar: { bcid: 'rationalizedCodabar', encode: (opts) => rationalizedCodabar(opts, drawingSVG()) },
```

ESLint

[error] Replace `·bcid:·'rationalizedCodabar',·encode:·(opts)·=>·rationalizedCodabar(opts,·drawingSVG())·` with `⏎↹↹bcid:·'rationalizedCodabar',⏎↹↹encode:·(opts)·=>·rationalizedCodabar(opts,·drawingSVG()),⏎↹`

(prettier/prettier)